LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -372,9 +378,9 @@ function getGroupList() {
         url: "../api/group/members",
         dataType: "json",
         success: function(res) {
+            console.log(res);
```

Comment:
Maybe want to remove `console.log`

Reply:
Found some more spots to clean up too; thanks for checking